torchci/pages/api/drci/drci.ts

```diff
@@ -101,93 +101,97 @@ export async function updateDrciComments(
     {};
 
   await forAllPRs(workflowsByPR, async (pr_info: PRandJobs) => {
-    const labels = await fetchIssueLabels(
-      octokit,
-      pr_info.owner,
-      pr_info.repo,
-      pr_info.pr_number
-    );
+    try {
```

Comment:
How about leaving this function as is (or perhaps renaming to `forAllPRs_` and create `forAllPRs` that wraps previos call with `try { } catch()`? Or it does not work like that with async functions?